utilities/pipelines/resourcePublish/Publish-ModuleToNuget.ps1

```diff
@@ -0,0 +1,524 @@
+
```

Comment:
Did we also add that to the publish step?

Reply:
Removed. We used it for the Hydration as it was a .psd PowerShell module and we offered the option in the pipeline to publish there too. At least until we merged the ADO pipelines we don't need to keep it